docs/content/documentation/systems/Materials/tensor_mechanics/ComputeIsotropicElasticityTensor.md

```diff
@@ -1,10 +1,35 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
-# ComputeIsotropicElasticityTensor
+# Compute Isotropic Elasticity Tensor
 !syntax description /Materials/ComputeIsotropicElasticityTensor
 
+## Description
+The material `ComputeIsotropicElasticityTensor` builds the isotropic elasticity (stiffness) tensor with two user provided elastic constants.
+`ComputeIsotropicElasticityTensor` also rotates the elasticity tensor during the initial time step only; this class does not rotate the elasticity tensor during the simulation.
```

Comment:
I think I'd strike the part about rotation here because rotating an isotropic tensor doesn't change it.